src/config/include/config/final_chain.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
```

Comment:
Uff, please change the name of the file, because now we have 2 files with the same name

Reply:
why is it a problem?

Reply:
I just find it confusing to have 2 files with the name `final_chain.hpp` when one is actually config for another one

Reply:
it is located in another directory and a lib. So you will see `config/final_chain.hpp`. Not sure that `config/final_chain_config.hpp` is better way to do it

Reply:
I think better way would be to have config per library and not library that consist of all configs as libraries should be independent

Reply:
I agree with Matus that every libraries should have it's own config as these libraries should (in ideal world) be relatively independent. Why should config for final_chain lib be in different lib ?

Reply:
That things is causing circular dependencies because that configs are needed in other libs and sometimes depends on each other. We still have a lot of it in project, I just started to deal with it in that PR